### PR TITLE
Project/login

### PR DESCRIPTION
**Pull request by NikolaSavic1709**
- Stored user data in PostgreSQL with hashed password and salt
- Authentication is implemented using JWT
- Some requests are only terminated if you have authorization header
- Filled database with few users

### CODE REVIEW

File: mini_zanzibar/internal/server/login.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt/v5"
+	"miniZanzibar/internal/database"
+	"net/http"
+	"strings"
+	"time"
+)
+
+var jwtSecret = []byte("secret_key")
```
Comment (ili0n) [inline mini_zanzibar/internal/server/login.go:14]: Make secret longer (min 512bit)

**Review by ili0n — CHANGES_REQUESTED** [review]
Fix jwt secret length.
Everything else is good.

**Review by ili0n — APPROVED** [review]
LGTM

**Comment by jovannajdovski** [comment]
Most of the security requirements for authentication are met. Looks good :1st_place_medal:

**Pull request merged**